docs/understanding-your-pipeline/failed-events/index.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: "Understanding failed events"
-date: "2020-02-15"
-sidebar_position: 1
+sidebar_label: "Failed events"
+sidebar_position: 4
+description: "Failed events represent data that did not pass validation or otherwise failed to be processed"
```

Comment:
Should we have another "Terminology" block here referencing "bad rows"?

Should the reference/link to recovery be a bit more prominent? (Hidden in a `<details>` atm I think?)